starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/ItemObject.java

```diff
@@ -0,0 +1,48 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+/**
+ * Created by KorbBookProReturns on 10/25/16.
+ */
+
+public class ItemObject {
+    private String mName, mDescription, mPrice, mType;
+
+    public ItemObject(String name, String description, String price, String type){
+        mName = name;
+        mDescription = description;
+        mPrice = price;
+        mType = type;
+    }
+
+    public String getName() {
+        return mName;
+    }
+
+    public String getDescription() {
+        return mDescription;
+    }
+
+    public String getPrice() {
+        return mPrice;
+    }
+
+    public String getType() {
+        return mType;
+    }
+
+    public void setName(String name) {
+        mName = name;
```

Comment:
Totally fine to have setters here, but I just want to make sure you know you don't always need to have setters. Sometimes you don't want an object's values to change after it's created, so you might prefer not to provide any setters.